plugins/opentelemetry/wiring.go

```diff
@@ -1,25 +1,68 @@
-// Package opentelemetry provides two plugins:
-// (i)  a plugin to generate and include an opentelemetry collector instance in a Blueprint application
-// (ii) provides a modifier plugin to wrap the service with an OpenTelemetry wrapper to generate OT compatible traces/logs.
+// Package opentelemetry provides three plugins:
 //
-// The package provides an in-memory trace exporter implementation and a go-client for generating traces on both the server and client side.
-// The generated clients handle context propagation correctly on both the server and client sides.
+//  1. a plugin to wrap the service with an OpenTelemetry wrapper to generate OT compatible traces by starting and stopping client spans for remote calls between services and correctly propagating context between services. The traces are then exported to a collector client such as jaeger or zipkin.
+//  2. a plugin to install an opentelemetry trace-based event logger for a go process. The logger adds all the logs as events to the current active span. If no active span exists, then no events are logged.
 //
-// The applications must use a backend.Tracer (runtime/core/backend) as the interface in the workflow.
+// Once the application is instrumented with these plugins, traces will be generated and collected by the selected collector. Traces will consist of spans generated by Plugin 1 with each span consisting of log events generated by Plugin 2.
+//
+// # Artifacts Generated
+//
+//  1. The package generates client and server side wrappers for instrumented services that contain opentelemetry instrumentation (context propagation, creation of spans). The generated clients handle context propagation correctly on both the server and client sides.
```

Comment:
Mention that the implementation of the logger is in the [runtime/plugins/opentelemetry] package and if the logger is configured, that's what gets used.